### ISSUE TITLE
Considering adding exclude_from_weight_decay or specified decay parameter list to bertadam(or other related optimizers)?

### ISSUE THREAD

**Issue by zburning** [dmlc/gluon-nlp#1088]
## Description
For bert-style model training, usually we will need to use weight decay. But we may not want to include some parameters to weight decay, e.g. layernorm. 
For example, in [BERT](https://github.com/google-research/bert/blob/cc7051dc592802f501e8a6f71f8fb3cf9de95dc9/optimization.py#L65), they have a argument exclude_from_weight_decay. In the current [tensorflow](https://github.com/tensorflow/tensorflow/blob/r1.15/tensorflow/contrib/opt/python/training/weight_decay_optimizers.py#L108), they implement a DecoupledWeightDecayExtension class to inherit from and specified it as decay_var_list.
I did not see if we have such a choice, do you guys think it is necessary do add it?

## References
- list reference and related literature
- list known implementations

**Comment by eric-haibin-lin** [comment]
weight decay multiplier is set via the gluon parameter API: https://github.com/dmlc/gluon-nlp/blob/master/scripts/bert/finetune_squad.py#L375-L377 is this not sufficient?

**Comment by zburning** [comment]
Great, thank you!
